packages/orchestration/src/index.ts

```diff
@@ -8,6 +8,7 @@ export type {
   ModuleConfigs,
   MaskingModuleConfig,
   MaskingProviderConfig,
+  // TODO: decide whether these types need to be renamed
```

Comment:
The types below (and others) are not compatible with our style guide. We should make a decision on how to handle this. I will asses where those types are used (return types, parameters) and what the complexity of doing that is. Theoretically they would need to be adjusted all over. Generally, I would prefer it to keep the API consistent.